bin/cml-runner.js

```diff
@@ -234,12 +235,34 @@ const runLocal = async (opts) => {
     } else if (log && log.status === 'job_ended') {
       const { job } = log;
 
+      const waitCompletedPipelineJobs = () => {
+        return new Promise((resolve, reject) => {
+          try {
+            if (RUNNER_JOBS_RUNNING.length === 1) {
+              resolve([RUNNER_JOBS_RUNNING[0].id]);
+              return;
+            }
+
+            const watcher = setInterval(async () => {
+              const jobs = (
+                await cml.pipelineJobs({ jobs: RUNNER_JOBS_RUNNING })
```

Comment:
The GitLab driver does not have `pipelineJobs` 🤔

Reply:
Its not needed for it

Reply:
After https://github.com/iterative/cml/pull/653/commits/082fcba905e2708de9d013af9a222e6fc2a98438 it won't throw a TypeError when calling an undefined function. Still, I suspect that it's going to throw the `not implemented` error you introduce.

Should we call `waitCompletedPipelineJobs` only if `driver === 'github'` or something?

Reply:
No, actually that will never be called in Gitlab. We maintain it to give coherence and to establish the interface. n Gitalb that will never ever be called

Reply:
Let me clarify. In Gitlab the logs comes with the jobID thing that does not happen in Github and we have to gather the jobs to check the completion of the id we knew

Reply:
> _No, actually that will never be called in Gitlab_ <sup>[citation needed]</sup>

Is there any guard code to prevent that? 🤔

Reply:
> Is there any guard code to prevent that? 🤔

the existance of the job.id

Reply:
```js
const jobs = job ? [job] : await waitCompletedPipelineJobs();
```

Reply:
As  say in GL that id is known always in the log

Reply:
👌🏼  Thanks!